backend/api/user.go

```diff
@@ -36,6 +73,7 @@ func returnAllUsers(c echo.Context) error {
 func init() {
 	log.Info("Running user init")
 
+	registerRoute(route{method: http.MethodGet, path: "/user/:id", handler: user})
```

Comment:
I believe the handler for this should be the GetUserByID function rather than user.

Reply:
Resolved in new push,